packages/amplify-cli/src/commands/diagnose.ts

```diff
@@ -1,23 +1,21 @@
-import {
-  stateManager, pathManager, NotInitializedError, spinner, DiagnoseReportUploadError,
-} from 'amplify-cli-core';
-import archiver from 'archiver';
-import * as fs from 'fs-extra';
-import * as path from 'path';
-import fetch from 'node-fetch';
-import { Redactor, stringMasker } from 'amplify-cli-logger';
-import columnify from 'columnify';
-import * as _ from 'lodash';
-import os from 'os';
-import { v4 } from 'uuid';
-import { prompter, printer } from 'amplify-prompts';
-import { collectFiles } from './helpers/collect-files';
-import { encryptBuffer, encryptKey, createHashedIdentifier } from './helpers/encryption-helpers';
-import { UsageDataPayload } from '../domain/amplify-usageData/UsageDataPayload';
-import { DebugConfig } from '../app-config/debug-config';
-import { isHeadlessCommand } from '../utils/headless-input-utils';
-import { Context } from '../domain/context';
-import { reporterEndpoint } from './helpers/reporter-apis';
+import { stateManager, pathManager, NotInitializedError, spinner, DiagnoseReportUploadError } from "amplify-cli-core";
```

Comment:
The eslint config should prefer single quotes, do you have a vscode setting that caused this?

Reply:
I am not sure, my linter keeps crashing on my machine

Reply:
fixed